sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Release History
 
-## 7.0.0-beta.2 (Unreleased)
-
+## 7.0.0-beta.2 (2020-05-07)
+
+- Add support for receiving messages from specific sessions
+- Add support for processing messages from multiple sessions
+- Add missing schedule and cancel APIs in ServiceBusSenderClient
```

Comment:
I thought they were missing only in the sync client right?

Also, "schedule and cancel" doesnt mean much on their own

> Add missing APIs for scheduling to send messages at a later time and cancelling such scheduled messages

Reply:
Yes, the async client is ServiceBusSenderAsyncClient